src/main/java/com/direwolf20/buildinggadgets/common/building/modes/TargetedAxisChasingMode.java

```diff
@@ -0,0 +1,45 @@
+package com.direwolf20.buildinggadgets.common.building.modes;
+
+import com.direwolf20.buildinggadgets.common.BuildingGadgets;
+import com.direwolf20.buildinggadgets.common.building.IPlacementSequence;
+import com.direwolf20.buildinggadgets.common.building.IValidatorFactory;
+import com.direwolf20.buildinggadgets.common.building.placement.Column;
+import com.direwolf20.buildinggadgets.common.building.placement.ExclusiveAxisChasing;
+import com.direwolf20.buildinggadgets.common.tools.IAtopSupport;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.BlockPos;
+
+/**
+ * <p>Formal "Build to Me"</p>
```

Comment:
```suggestion
 * <p>Former "Build to Me"</p>
```

Reply:
The "Former" here means "Old name", and "Formal" does not suit the need.